Source/control.cpp

```diff
@@ -2132,28 +2132,24 @@ void CheckLvlBtn()
 	if (!lvlbtndown && MouseX >= 40 && MouseX <= 81 && MouseY >= 313 && MouseY <= 335)
 		lvlbtndown = 1;
 }
-// 4B851C: using guessed type int lvlbtndown;
 
 void ReleaseLvlBtn()
 {
 	if (MouseX >= 40 && MouseX <= 81 && MouseY >= 313 && MouseY <= 335)
 		chrflag = 1;
 	lvlbtndown = 0;
 }
-// 4B851C: using guessed type int lvlbtndown;
 
 void DrawLevelUpIcon()
 {
-	int v0; // esi
+	int nCel;
 
 	if (!stextflag) {
-		v0 = (lvlbtndown != 0) + 2;
+		nCel = (lvlbtndown != 0) + 2;
 		ADD_PlrStringXY(0, 303, 120, "Level Up", 0);
-		CelDecodeOnly(104, 495, (BYTE *)pChrButtons, v0, 41);
+		CelDecodeOnly(104, 495, (BYTE *)pChrButtons, nCel, 41);

```

Comment:
Shouldn't `pChrButtons` be actually `BYTE*` since `LoadFileInMem` returns this type? Then this cast could be removed

Reply:
More spring cleaning I guess :)